### PR TITLE
Added ticket announcement blog for workshops and devsprints

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @palnabarun @vipulgupta2048 @sayanchowdhury

### CODE REVIEW

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+[**Go get your tickets, available now on our website**](https://in.pycon.org/2020/#ticket)
+
+
+## Devsprints
+
+PyCon India 2020 Devsprints will happen on 4th and 5th October. We are happy to announce the devsprint tickets are now open. 
+
+You don't need to have a conference ticket to participate in the devsprints, anyone can buy the tickets. **The devsprint tickets will be free this year.**
+
+To know more about devsprints, check out our [blog on devsprints](). You can find the list of projects on our [wiki page](https://github.com/pythonindia/inpycon2020/wiki/List-of-PyCon-India-2020-Projects). If you are a contributor/maintainer for a project and would like to represent your project in the devsprints, please refer to our [devsprints page](https://github.com/pythonindia/inpycon2020/wiki/PyCon-India-2020-Devsprint) to add your project to our devsprints.
```
Comment (Sc0rpi0n101) [inline content/blog/2020-workshop-devsprints-tickets.md:20]: Need the blog link here, but it's not been merged yet

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Title: Workshop and Devsprint tickets are now open
+Date: 2020-09-15 21:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-workshop-devsprint-tickets
```
Comment (palnabarun) [inline content/blog/2020-workshop-devsprints-tickets.md:5]:
```suggestion
Slug: workshop-devsprint-tickets
```

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Summary: PyCon India 2020 workshop and devsprint tickets are now open.
+
+
+We know you all have been waiting for this eagerly. Don't worry, you don't have to wait anymore! The Workshop and devsprint tickets for PyCon India are now open. 
+
+[**Go get your tickets, available now on our website**](https://in.pycon.org/2020/#ticket)
```
Comment (palnabarun) [inline content/blog/2020-workshop-devsprints-tickets.md:11]: We are launching the blog tonight but tickets would be available tomorrow 5PM.

I maybe wrong there. cc: @sayanchowdhury

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Title: Workshop and Devsprint tickets are now open
+Date: 2020-09-15 21:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-workshop-devsprint-tickets
+Summary: PyCon India 2020 workshop and devsprint tickets are now open.
+
+
+We know you all have been waiting for this eagerly. Don't worry, you don't have to wait anymore! The Workshop and devsprint tickets for PyCon India are now open.
```
Comment (sayanchowdhury) [inline content/blog/2020-workshop-devsprints-tickets.md:9]: This is not the open post but going to open at 5PM tomorrow.

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+Title: Workshop and Devsprint tickets are now open
+Date: 2020-09-15 21:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-workshop-devsprint-tickets
+Summary: PyCon India 2020 workshop and devsprint tickets are now open.
```
Comment (sayanchowdhury) [inline content/blog/2020-workshop-devsprints-tickets.md:6]: This is not the open post but going to open at 5PM tomorrow.

File: content/blog/2020-workshop-devsprints-tickets.md (hunk 1 above)
Comment (palnabarun) [reply content/blog/2020-workshop-devsprints-tickets.md:20]: You can prefill the link using the slug in that unmerged article, or remove the reference altogether.

To that effect, will PR merge in before this gets in?

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+PyCon India 2020 Devsprints will happen on 4th and 5th October. We are happy to announce the devsprint tickets are now open. 
+
+You don't need to have a conference ticket to participate in the devsprints, anyone can buy the tickets. **The devsprint tickets will be free this year.**
+
+To know more about devsprints, check out our [blog on devsprints](). You can find the list of projects on our [wiki page](https://github.com/pythonindia/inpycon2020/wiki/List-of-PyCon-India-2020-Projects). If you are a contributor/maintainer for a project and would like to represent your project in the devsprints, please refer to our [devsprints page](https://github.com/pythonindia/inpycon2020/wiki/PyCon-India-2020-Devsprint) to add your project to our devsprints.
+
+## Workshops
+
+The workshops for this year will be taking place on October 4. Each workshop is 2.5 hours duration. There are a total of 6 parallel tracks, with two workshops in each track, one in the morning and one in the afternoon. You can attend a maximum of two workshops, morning and afternoon.
```
Comment (sayanchowdhury) [inline content/blog/2020-workshop-devsprints-tickets.md:24]: 4 parallel track. You can attend any number of workshops, there is one ticket for workshop, and not individual tickets

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+**You have to buy a separate ticket for each individual workshop**
+
+## Attending Multiple Workshops
```
Comment (sayanchowdhury) [inline content/blog/2020-workshop-devsprints-tickets.md:28]: This can be totally gone, and explained in the other section.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: content/blog/2020-workshop-devsprints-tickets.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+You can attend more than one workshop. But since the workshops will be run synchronously, you will only be able to attend one workshop at a time. 
+
+You can buy a maximum of two workshop tickets, one for the morning session, and one for the afternoon session. 
+
+## Attending Workshops and Devsprint
```
Comment (sayanchowdhury) [inline content/blog/2020-workshop-devsprints-tickets.md:34]: Also adjust the text here.

**Review by palnabarun — APPROVED** [review]

**Pull request merged**